bundles/org.openhab.binding.adorne/src/main/java/org/openhab/binding/adorne/internal/handler/AdorneHubHandler.java

```diff
@@ -105,16 +102,15 @@ public void dispose() {
     @Override
     public void stateChangeNotify(int zoneId, boolean onOff, int brightness) {
         logger.debug("State changed (zoneId:{} onOff:{} brightness:{})", zoneId, onOff, brightness);
-        AdorneSwitchHandler thingHandler;
-        for (Thing thing : bridge.getThings()) {
-            thingHandler = (AdorneSwitchHandler) thing.getHandler();
+        getThing().getThings().forEach(thing -> {
+            AdorneSwitchHandler thingHandler = (AdorneSwitchHandler) thing.getHandler();
             if (thingHandler != null && thingHandler.getZoneId() == zoneId) {
                 thingHandler.updateState(CHANNEL_POWER, onOff ? OnOffType.ON : OnOffType.OFF);
```

Comment:
```suggestion
                thingHandler.updateState(CHANNEL_POWER, OnOffType.from(onOff));
```

Reply:
I originally had the for loop, but changed it to forEach as per your suggestion above for zoneIDs. Switched this one back. 
Nice shortcut for the onOff - done.

Reply:
Did you forget to push these changes?

Reply:
sorry, didn't push yet - unresolving comment.

Reply:
Pushed